{"name": "test_attach_photo", "status": "broken", "statusDetails": {"message": "OSError: [Errno 22] Invalid argument: 'G:\\\\Workspace\\\\pyworkspace\\resouces\\\\png1.jpeg'", "trace": "def test_attach_photo():\n        #当传文件时 要用到file方法\n>       allure.attach.file(\"G:\\Workspace\\pyworkspace\\resouces\\png1.jpeg\",name=\"这是一个图片\",attachment_type=allure.attachment_type.JPG)\n\ntest_attach.py:11: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\ne:\\program files\\python3.7\\lib\\site-packages\\allure_commons\\_allure.py:182: in file\n    plugin_manager.hook.attach_file(source=source, name=name, attachment_type=attachment_type, extension=extension)\ne:\\program files\\python3.7\\lib\\site-packages\\pluggy\\hooks.py:286: in __call__\n    return self._hookexec(self, self.get_hookimpls(), kwargs)\ne:\\program files\\python3.7\\lib\\site-packages\\pluggy\\manager.py:93: in _hookexec\n    return self._inner_hookexec(hook, methods, kwargs)\ne:\\program files\\python3.7\\lib\\site-packages\\pluggy\\manager.py:87: in <lambda>\n    firstresult=hook.spec.opts.get(\"firstresult\") if hook.spec else False,\ne:\\program files\\python3.7\\lib\\site-packages\\allure_pytest\\listener.py:226: in attach_file\n    self.allure_logger.attach_file(uuid4(), source, name=name, attachment_type=attachment_type, extension=extension)\ne:\\program files\\python3.7\\lib\\site-packages\\allure_commons\\reporter.py:114: in attach_file\n    plugin_manager.hook.report_attached_file(source=source, file_name=file_name)\ne:\\program files\\python3.7\\lib\\site-packages\\pluggy\\hooks.py:286: in __call__\n    return self._hookexec(self, self.get_hookimpls(), kwargs)\ne:\\program files\\python3.7\\lib\\site-packages\\pluggy\\manager.py:93: in _hookexec\n    return self._inner_hookexec(hook, methods, kwargs)\ne:\\program files\\python3.7\\lib\\site-packages\\pluggy\\manager.py:87: in <lambda>\n    firstresult=hook.spec.opts.get(\"firstresult\") if hook.spec else False,\ne:\\program files\\python3.7\\lib\\site-packages\\allure_commons\\logger.py:53: in report_attached_file\n    shutil.copy2(source, destination)\ne:\\program files\\python3.7\\lib\\shutil.py:257: in copy2\n    copyfile(src, dst, follow_symlinks=follow_symlinks)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nsrc = 'G:\\\\Workspace\\\\pyworkspace\\resouces\\\\png1.jpeg', dst = 'G:\\\\Workspace\\\\pyworkspace\\\\testbag\\\\result\\\\7\\\\a0d21df0-691d-4bf4-be63-54097f736653-attachment.jpg'\n\n    def copyfile(src, dst, *, follow_symlinks=True):\n        \"\"\"Copy data from src to dst.\n    \n        If follow_symlinks is not set and src is a symbolic link, a new\n        symlink will be created instead of copying the file it points to.\n    \n        \"\"\"\n        if _samefile(src, dst):\n            raise SameFileError(\"{!r} and {!r} are the same file\".format(src, dst))\n    \n        for fn in [src, dst]:\n            try:\n                st = os.stat(fn)\n            except OSError:\n                # File most likely does not exist\n                pass\n            else:\n                # XXX What about other special files? (sockets, devices...)\n                if stat.S_ISFIFO(st.st_mode):\n                    raise SpecialFileError(\"`%s` is a named pipe\" % fn)\n    \n        if not follow_symlinks and os.path.islink(src):\n            os.symlink(os.readlink(src), dst)\n        else:\n>           with open(src, 'rb') as fsrc:\nE           OSError: [Errno 22] Invalid argument: 'G:\\\\Workspace\\\\pyworkspace\\resouces\\\\png1.jpeg'\n\ne:\\program files\\python3.7\\lib\\shutil.py:120: OSError"}, "attachments": [{"name": "这是一个图片", "source": "a0d21df0-691d-4bf4-be63-54097f736653-attachment.jpg", "type": "image/jpg"}], "start": 1637992962390, "stop": 1637992962391, "uuid": "7dc8489a-98ac-4d03-8048-040067f82c31", "historyId": "c96a6a5db98e26f72af10e1cd5f5d6c7", "testCaseId": "732ebcff52a5b38f2410c486bf5bc0f1", "fullName": "test_attach#test_attach_photo", "labels": [{"name": "suite", "value": "test_attach"}, {"name": "host", "value": "LAPTOP-QMS0DV6U"}, {"name": "thread", "value": "15688-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_attach"}]}